libs/wire-api/src/Wire/API/Routes/Named.hs

```diff
@@ -33,7 +33,9 @@ import Servant.Client.Core (clientIn)
 import Servant.OpenApi
 
 -- | See http://docs.wire.com/developer/developer/servant.html#named-and-internal-route-ids-in-swagger
-newtype Named name x = Named {unnamed :: x}
+type Named name = UntypedNamed (IsStronglyTyped name)
+
+newtype UntypedNamed name x = Named {unnamed :: x}
```

Comment:
I think some comments explaining what and why here might be good for posterity :)